bin/stacks/routing-lambda-stack.ts

```diff
@@ -100,7 +100,7 @@ export class RoutingLambdaStack extends cdk.NestedStack {
       // 11/8/23: URA currently calls the Routing API with a timeout of 10 seconds.
       // Set this lambda's timeout to be slightly lower to give them time to
       // log the response in the event of a failure on our end.
-      timeout: cdk.Duration.seconds(9),
+      timeout: cdk.Duration.seconds(19),
```

Comment:
let's not merge this.

I do see our cold start issue is starting to affect our local dev experience, as well as beta integ-tests. I actually spent some time investigating, and found out that the cold start is due to the routing-api S2 pool caching per uniswap subgraph version. Recently with more chains support, this does affect routing-api cold start.

I can log a ticket sometime today to describe the issue.

Reply:
Sorry that's for debug only. Reverted that change.